tests/ui_tests/app/page/common/table/page.py

```diff
@@ -56,8 +57,9 @@ def wait_rows_change(self):
         yield
 
         def wait_scroll():
-            assert len(self.get_all_rows()) != current_amount
+            assert len(self.get_all_rows()) != current_amount, "Pade has not changed"
```

Comment:
Better to specify what kind of change was expected. For example "Amount of rows on the page hasn't changed"

Reply:
fixed